cmd/kubeadm/app/master/manifests.go

```diff
@@ -385,7 +386,7 @@ func getControllerManagerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted
 	command = append(getComponentBaseCommand(controllerManager),
 		"--address=127.0.0.1",
 		"--leader-elect",
-		"--master=127.0.0.1:8080",
+		"--kubeconfig="+path.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, kubeadmconstants.ControllerManagerKubeConfigFileName),
```

Comment:
Cool to make you happy :)

I still think that the `path.Join` expression is terrible, but some first-time contributor said we wanted to claim to clean those up for us, so until I know whether he will do it or not I'll wait for it